contracts/relayer/RelayerAragonAppWithVolatileSender.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.4.24;
+
+import "./BaseRelayer.sol";
+
+
+contract RelayerAragonAppWithVolatileSender is BaseRelayer {
+    function exec(address from, uint256 nonce, bytes calldata, bytes signature) external refundGas auth(OFF_CHAIN_RELAYER_SERVICE_ROLE) {
+        assertValidTransaction(from, nonce, calldata, signature);
+
+        setVolatileStorageSender(from);
+        setUsedNonce(from, nonce, true);
+
+        bool success = address(this).call(calldata);
```

Comment:
Same comment as above about the external app.

Reply:
Answer above